src/js/yeldaWebchatInjector.js

```diff
@@ -49,7 +49,8 @@ class YeldaChat {
     this.webChatContainer.setAttribute('class', classList)
     this.parentContainer.appendChild(this.webChatContainer)
 
-    if (this.parentContainer === document.body) {
+    // Add assistant image if the webchat can be closed
+    if (!data.hasOwnProperty('canBeClosed') || data.canBeClosed) {
       this.addAssistantImage(data)
     }
   }
```

Comment:
It is related to the fact that parentContainerId forced canBeClosed to false
With the initial condition `this.parentContainer === document.body` and canBeClosed set to true, I could close the webchat but couldn't open it again because parentContainer was not document.body therefore, the assistant image containing the 'openChat' event was not created